docs/source/testing/testing.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: Integration testing
+description: Utilities for testing Apollo Server
+---
+
+import { MultiCodeBlock } from 'gatsby-theme-apollo-docs';
+
+Apollo Server uses a [multi-step request pipeline](../integrations/plugins/#request-lifecycle-event-flow) to validate and execute incoming GraphQL operations. This pipeline supports integration with custom plugins at each step, which can affect an operation's execution. Because of this, it's important to perform **integration tests** with a variety of operations to ensure your request pipeline works as expected.
+
+There are two main options for integration testing with Apollo Server:
+* Using `ApolloServer`'s `executeOperation` method.
+* Setting up an HTTP client to query your server.
+
+## Testing using `executeOperation`
+
+Apollo Server's `executeOperation` method enables you to run operations through the request pipeline _without_ sending an HTTP request.
+
+The [`executeOperation` method](../api/apollo-server/#executeoperation) accepts the following arguments:
+* An object containing configuration options describing the GraphQL operation to preform.
```

Comment:
"perform" not "prefrom"

but in general across this PR can you move away from the concept of "configuration options" for this sort of thing? This isn't describing the "configuration" of the server — it's describing the particular operation that we're trying to execute on a server that has already been configured.

Reply:
Thank you for the typo catch (and for helping me realize my spell-checking extension was bugging out 😅). 

You raise a very good point, I think omitting the word "configuration"  would be the easiest way to go about this! Simply stating:  "An object containing the options describing the GraphQL operation to perform."  

If that looks good to you, I can remove the corresponding mentions of "configuration" from the `executeOperation` section of the API docs as well.

Reply:
Sure, though I am not sure I'd even call it options in this case. I think of "options" (which in JS isn't a technical part of the language like in other languages where you can have named function parameters) as really being about choices being made in how you are calling the function ("which optional features are you using") whereas this argument here really feels like *data*.